src/components/widgets/RaceTrackerMap/RaceTrackerMap.tsx

```diff
@@ -1,28 +1,46 @@
 import * as _ from 'lodash';
 import * as React from "react";
 
+import WebMercatorViewport from 'viewport-mercator-project';
+import bbox from '@turf/bbox';
+import * as turf from '@turf/turf'
+
 import ReactMapGL, {
   NavigationControl,
-  FlyToInterpolator
+  FlyToInterpolator,
+  // LinearInterpolator
 } from "react-map-gl";
 import {
   defaultMapStyle,
   pointLayer
 } from "./MapStyle";
-import {fromJS} from "immutable";
-import {Img} from 'components/Img/Img';
+import { fromJS } from "immutable";
+import { Img } from 'components/Img/Img';
 import * as trackLeaderService from 'services/trackLeaders';
-import {takeUntil} from 'rxjs/operators';
-import {Subject} from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
 
+import imgDaveMinusPin from 'assets/images/personPinMinus.png';
 import imgPersonPin from 'assets/images/personPin.png';
+import imgDavePlusPin from 'assets/images/personPinPlus.png';
+
 
 import styles from './RaceTrackerMap.module.css';
+// import { number } from 'prop-types';
+
 
 
 
 const TOKEN = "pk.eyJ1IjoicnNiYXVtYW5uIiwiYSI6ImNqdzg5aWxkYzF1azI0OW5uaWVmazhleXUifQ.XAm1dRGmXuRAMSQm0TJKXg";
```

Comment:
Update to `pk.eyJ1IjoicnNiYXVtYW5uIiwiYSI6ImNqd281d2M3bzF4MHczeXA5dm9xdXg4bjAifQ._vwACgp12mC4RBpuSYl1CA`

Reply:
@ryanbaumann @Rayhan96Shuvo I will do this and push to master. Do not do this in this branch.

Reply:
no sweat.  Just opened this PR to give feedback on a few q's that @Rayhan96Shuvo had in Slack